wrapanapi/systems/virtualcenter.py

```diff
@@ -955,6 +955,17 @@ def _search_folders_for_vm(self, name):
 
         return vm
 
+    def create_folder(self, folder_name):
+        """Create a Vm folder under Datacenter"""
+        dc = self.get_obj(vim.Datacenter, "Datacenter")
+        folder = dc.vmFolder.CreateFolder(folder_name)
+        return folder
+
+    def destroy_folder(self, folder_name):
```

Comment:
**Suggestion:** 
@ganeshhubale In my opinion the folder name should be `delete_folder`.

Reply:
I'd leave this name. it's ok here

Reply:
Just thought in terms of Create-Delete :smile: